scripts/build

```diff
@@ -520,25 +509,25 @@ if [ "$TARGET" = "target" -o "$TARGET" = "init" ]; then
     find $INSTALL -type d -exec rmdir -p {} \; 2>/dev/null || :
 
     if [ ! "${BUILD_WITH_DEBUG}" = "yes" ]; then
-      $STRIP `find $INSTALL \
+      $STRIP $(find $INSTALL \
         -type f -name "*.so*" \
         ! -name "ld-*.so" \
         ! -name "libc-*.so" \
         ! -name "libpthread-*.so" \
         ! -name "libthread_db-*so" \
-        2>/dev/null` 2>/dev/null || :
+        2>/dev/null) 2>/dev/null || :
         if [ "$TARGET" = "init" ]; then
-          $STRIP `find $INSTALL -type f -name "*.so*" 2>/dev/null` 2>/dev/null || :
+          $STRIP $(find $INSTALL -type f -name "*.so*" 2>/dev/null) 2>/dev/null || :
         fi
-        $STRIP `find $INSTALL/bin $INSTALL/usr/bin $INSTALL/sbin $INSTALL/usr/sbin \
-          -type f -executable 2>/dev/null` 2>/dev/null || :
+        $STRIP $(find $INSTALL/bin $INSTALL/usr/bin $INSTALL/sbin $INSTALL/usr/sbin \
+          -type f -executable 2>/dev/null) 2>/dev/null || :
     fi
   fi
 fi
 
 cd $ROOT
 
-for i in `find $SYSROOT_PREFIX/usr/lib/ -name "*.la" 2>/dev/null`; do \
+for i in $(find $SYSROOT_PREFIX/usr/lib/ -name "*.la" 2>/dev/null); do \
   $SED "s:\(['= ]\)/usr:\\1$SYSROOT_PREFIX/usr:g" $i; \
 done
```

Comment:
While you're at it, would you mind removing the line continuation symbols after `do` and `$SED` (which also has an errant semi-colon) as they're not required - thanks!

Also, I think it would be better to add `-e` and `-i` to $SED, ie.:
```
   $SED -e "s:\(['= ]\)/usr:\\1$SYSROOT_PREFIX/usr:g" -i $i 
``

Reply:
`$SED` gets expanded to `sed -i` (defined in config/path). A quick look at the code shows a fairly hit or miss of packages using `$SED` versus calling sed on their own. Opinion on retiring `$SED`?

Reply:
I see you've removed the `\` (line continuation) - thanks. Any concerns about removing the `;` at the end of the `$SED` - it shouldn't be required with `sed` or `$SED`...